### ISSUE TITLE
asking for dadesets

### ISSUE THREAD

**Issue by BIG9MONKEY** [Lanyu0303/AFL#8]
Hi there, may I ask which version of the Steam and MovieLens datasets you are used for? Could you share the datasets you're working with? I would be very grateful!

**Comment by Lanyu0303** [comment]
Thank you for your interest in our work. We used the Steam and MovieLens datasets processed by LLaRA, which you can download from their repository: https://github.com/ljy0ustc/LLaRA

**Comment by BIG9MONKEY** [comment]
> Thank you for your interest in our work. We used the Steam and MovieLens datasets processed by LLaRA, which you can download from their repository: https://github.com/ljy0ustc/LLaRA感谢您对我们工作的关注。我们使用了由 LLaRA 处理的 Steam 和 MovieLens 数据集，您可以从其代码库下载：https://github.com/ljy0ustc/LLaRA

.


Thank you! The dataset has been found. I have a new question now. In your paper, you mentioned "divide the data into training, validation, and test sets in the ratio of 8:1:1,  In practice, the training set and validation set are used for the training and validation of
the recommendation agent’s recommendation model and the user agent’s reward model"， May I ask whether traditional recommendation models and LLM-based recommendation models both use the last, movielens, and steam datasets mentioned here for training?

**Comment by Lanyu0303** [comment]
Yes, each model will train a version for each dataset.
